.golangci.yaml

```diff
@@ -4,9 +4,6 @@ run:
 
   timeout: 5m
 
-  build-tags:
-    - aws
```

Comment:
Does this mean we're no longer checking the AWS code paths?

Reply:
No, the linters don't use build tags. I did a test to confirm by adding a bunch of if's which gocyclo picked up. Also checked goimports corrects code inside of the aws cloud provider.